### PR TITLE
Release v2.20.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,37 @@
...
+
+### Bug Fixes
+
+- **AWS API Gateway:**
+  - Correctly set `throttle` when `quota` missing ([#8780](https://github.com/serverless/serverless/pull/8780)) ([4a30bb1](https://github.com/serverless/serverless/commit/4a30bb1e5b36b52207e1bd3f3fc37e12878fb3b3)) ([Cem Enson](https://github.com/cemenson))
+  - Silence timeout warning for `async: true` ([#8748](https://github.com/serverless/serverless/issues/8748)) ([0384776](https://github.com/serverless/serverless/commit/03847769cd238824cbe9ea9fdec1889645081b17)) ([Igor Omelchenko](https://github.com/MEGApixel23))
+- **AWS Lambda:** Ensure function update works when image used ([#8786](https://github.com/serverless/serverless/issues/8786)) ([420e937](https://github.com/serverless/serverless/commit/420e93740f1e9bffc285559b2567379f550f28af)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **AWS CloudFront:** Fix check for deprecated CacheBehavior properties ([#8768](https://github.com/serverless/serverless/pull/8768)) ([c3a61e2](https://github.com/serverless/serverless/commit/c3a61e234bf73429b946e09121b48306e56e0ed5)) ([Ben Scholzen](https://github.com/DASPRiD))
+- **CLI:**
```
Comment (pgrzesik) [inline CHANGELOG.md:21]: @medikoo Do you think it makes sense to list these or should they just be squashed under `Seclude service config path resolution out of internals` in maintenance section?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,37 @@
...
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.20.0](https://github.com/serverless/serverless/compare/v2.19.0...v2.20.0) (2021-01-21)
+
+### Features
+
+- **AWS Lambda:**
+  - Add support for building Docker images ([#8725](https://github.com/serverless/serverless/issues/8725)) ([789c2e3](https://github.com/serverless/serverless/commit/789c2e35ab26b7e8dc0679f36110234fb899d57c)) ([Piotr Grzesik](https://github.com/pgrzesik))
+  - Add support for image config ([#8778](https://github.com/serverless/serverless/issues/8778)) ([9a55537](https://github.com/serverless/serverless/commit/9a5553742a3c3ebee03bfab5663a9183d5c228ba)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:** Support absolute path at "--config" ([#8793](https://github.com/serverless/serverless/pull/8793)) ([aea9f93](https://github.com/serverless/serverless/commit/aea9f9397c3806edda9958e324678af168b46b2f)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (pgrzesik) [inline CHANGELOG.md:12]: Should that be a feature or a fix? @medikoo

Comment (medikoo) [reply CHANGELOG.md:12]: Good question. I believe this should be skipped. As it's an improvement for refactor introduced with #8770). So it was an improvement in context of `master`, but it's not any improvement against v2.19.0 (and this CHANGELOG should be based against that)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,37 @@
...
+
+### Bug Fixes
+
+- **AWS API Gateway:**
+  - Correctly set `throttle` when `quota` missing ([#8780](https://github.com/serverless/serverless/pull/8780)) ([4a30bb1](https://github.com/serverless/serverless/commit/4a30bb1e5b36b52207e1bd3f3fc37e12878fb3b3)) ([Cem Enson](https://github.com/cemenson))
+  - Silence timeout warning for `async: true` ([#8748](https://github.com/serverless/serverless/issues/8748)) ([0384776](https://github.com/serverless/serverless/commit/03847769cd238824cbe9ea9fdec1889645081b17)) ([Igor Omelchenko](https://github.com/MEGApixel23))
+- **AWS Lambda:** Ensure function update works when image used ([#8786](https://github.com/serverless/serverless/issues/8786)) ([420e937](https://github.com/serverless/serverless/commit/420e93740f1e9bffc285559b2567379f550f28af)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **AWS CloudFront:** Fix check for deprecated CacheBehavior properties ([#8768](https://github.com/serverless/serverless/pull/8768)) ([c3a61e2](https://github.com/serverless/serverless/commit/c3a61e234bf73429b946e09121b48306e56e0ed5)) ([Ben Scholzen](https://github.com/DASPRiD))
```
Comment (medikoo) [inline CHANGELOG.md:20]: We may bacj quite `CacheBehavior`

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,37 @@
...
+  - Fix configuration of a new service in interactive setup ([#8770](https://github.com/serverless/serverless/pull/8770)) ([76fa62d](https://github.com/serverless/serverless/commit/76fa62da3b050260063f52cb0586f626ff6de018)) ([Mariusz Nowak](https://github.com/medikoo))
+
+### Maintenance Improvements
+
+- **CLI:** Seclude service config path resolution out of internals ([#8770](https://github.com/serverless/serverless/pull/8770)) ([b23bfdb](https://github.com/serverless/serverless/commit/b23bfdbf6ad915ec00fec562f8b75c40c44dd19d)) ([Mariusz Nowak](https://github.com/medikoo))
+- Mark functions async in `aws/customResources` and `aws/deploy` ([#8698](https://github.com/serverless/serverless/pull/8698)) ([c45f661](https://github.com/serverless/serverless/commit/c45f66117892e6f5948274288d7dda41f96dfe85)) ([ifitzsimmons](https://github.com/ifitzsimmons))
+- Improve dependencies handling ([#8770](https://github.com/serverless/serverless/pull/8770)) ([e3c64a3](https://github.com/serverless/serverless/commit/e3c64a3f36d72753b6dea5aa951c60f44251a071)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:29]: Let's remove that one (it's very minor and it was base for `Seclude service config path resolution out of internals` which is listed above)

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,37 @@
...
+
+### Templates
+
+- Add node version constraint to aws-nodejs-typescript ([#8776](https://github.com/serverless/serverless/pull/8776)) ([37d5f9e](https://github.com/serverless/serverless/commit/37d5f9e74024b54955eb4d503edfefcaf0b03444)) ([Frédéric Barthelet](https://github.com/fredericbarthelet))
+- Compilation target ES2019 in aws-nodejs-typescript ([#8774](https://github.com/serverless/serverless/pull/8774)) ([4469388](https://github.com/serverless/serverless/commit/4469388669d50193dedc6e2695789d24fe30a238)) ([Frédéric Barthelet](https://github.com/fredericbarthelet))
```
Comment (medikoo) [inline CHANGELOG.md:34]: Let's back quote template names

File: CHANGELOG.md (hunk 1 above)
Comment (medikoo) [reply CHANGELOG.md:21]: They're separate things, fixed in context of above.

I would keep them separate, but I would probably create a new dedicated scope for this as maybe `CLI Onboarding` (it's for our interactive CLI setup, which deserves separate context)

**Review by medikoo — CHANGES_REQUESTED** [review]
Thanks @pgrzesik! Good questions, see my answers

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**